validation-test/stdlib/String.swift

```diff
@@ -999,6 +1000,30 @@ StringTests.test(
 #endif
 }
 
+StringTests.test("String._fastCStringContents()")
+  .skip(.nativeRuntime("_NSContiguousString undefined without _runtime(_ObjC)"))
+  .code {
+#if _runtime(_ObjC)
+  // ASCII tests
+  for i in (0..<128) {
+    let cs = _NSContiguousString(String(UnicodeScalar(i)!)._core)
```

Comment:
It is not clear that this creates a string in the ASCII representation.  Could you assert the `isASCII` bit on the `StringCore`?